docs/development.md

```diff
@@ -28,7 +28,36 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 
 * To stop the dev server, click the "Terminate" icon on the Eclipse console.
 
-### Outside Eclipse
+### With IntelliJ
+
+> If this is your first time urnning the dev server, you will need to set up the required `Run Configuration`.
+
+#### Set up the Run Configuration
+
+1. In your `Event Log`, you should see this line: `Frameworks detected: Google App Engine, Web, JPA frameworks are detected in the project`. Click `Configure` and `OK` in the dialog box that appears.
+1. Go to `File → Project Structure...`.
+1. Under `Artifacts → Gradle : teammates.war (exploded)`, check `Include in project build`.
```

Comment:
Does `assemble` put the exploded app in a different folder for you too? Should change the location of the artifact rather than leaving two copies of the exploded app.